src/transformStateWithClones.js

```diff
@@ -64,8 +64,32 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const newState = Object.assign({}, state);
+
+  const changes = [];
+
+  for (const transform of transforms) {
```

Comment:
```suggestion
  for (const {operation, properties} of transforms) {
```
Можно использовать operation, properties вместо transform.operation, transform.properties